interface/src/app/gene/searchMany/gene-search-panel.tpl.html

```diff
@@ -14,4 +14,9 @@
       </search-result-table>
     </div>
   </div>
+  <div class="panel-footer">
+    <button type="button" class="btn btn-link" ng-click="switchToFew()">
+      Switch to searching with autocomplete.
+    </button>
```

Comment:
Ok, I played around with this layout and showed it to Jackie for a UX test, and I think it probably makes sense to keep this button/link here for consistency with the autocomplete view (which has it in the same place). Also, it is right below the "Search" button, so that might catch the users' eyes before they click Search.